torch/lib/c10d/ProcessGroupNCCL.hpp

```diff
@@ -289,6 +301,29 @@ class ProcessGroupNCCL : public ProcessGroup {
   // accordingly.
   void parseNcclBlockingWait();
 
+   // NCCL alltoall implemented using NCCL send/recv
+   ncclResult_t ncclAlltoall(
```

Comment:
do we also need to use ifdef to guard this?

Reply:
Moved to anonymous namespace in C++ file.